tests/testthat/test-import_instruments.R

```diff
@@ -3,11 +3,16 @@
 # Ray Balise and Gabriel Odom 
 # 2023-02-10
 
+library(testthat)
 
 ######  Setup  ######
 target <- structure(
   list(
-    record_id = c(1, 2, 3, 4, 5),
+    record_id = structure(
+      c(1, 2, 3, 4, 5), 
+      label = "Study ID", 
+      class = c("labelled", "numeric")
+    ), 
```

Comment:
I'm not sure why the new version of REDCapR produces this change.  But I haven't really studied what tidyREDCap does with the labelled class.

Reply:
labelled needs to be the last argument in the class list.  So we need a unit test to make sure that all is still good.  See the fix_class_bug() function  in make_instrument_auto.R.  It is a horrible kludge.  We need to look at the labeling function that REDCapTidieR is using to see if things could be simpler.